### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by ghost**
1st GitHub Task.

### CODE REVIEW

**Review by mhimanshu712 — CHANGES_REQUESTED** [review]
You should not have removed  the line "<!-Append your code above!-->"

File: docs/index.html [hunk 1]
```diff
@@ -137,9 +137,15 @@ <h3>Shreya Roy</h3>
 		</div>
 
 		
-		<!-- Append your code above!-->
```
Comment (mhimanshu712) [inline docs/index.html]: This was removed

Comment (ghost) [reply docs/index.html]: Sir, can you check now.

File: docs/index.html [hunk 2]
```diff
@@ -137,9 +137,20 @@ <h3>Shreya Roy</h3>
...
-
-
+		<<div class="col-sm-6 col-md-4">
```
Comment (mhimanshu712) [inline docs/index.html]: It shoud be **<div**   instead of **<<div**

**Review by mhimanshu712 — CHANGES_REQUESTED** [review]
Make required changes!

**Comment by mhimanshu712** [comment]
I removed it for you.

**Comment by mhimanshu712** [comment]
Great work 🎉
Merging the changes.

**Pull request merged**